react-client/src/components/MerlinChoice.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+class MerlinChoice extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+  }
+
+  render() {
+
+    return (
+      <div>
+
+      <Stats />
+
+      <Timer seconds={this.spyCount * 60}/>
+
+        <h3> Choose Merlin </h3>
+
+        A Submit button here
```

Comment:
actually here you would probably use the html form element, which includes a submit type